### ISSUE TITLE
More early help for first-time users

### ISSUE THREAD

**Issue by nbarbettini** [nbarbettini/little-aspnetcore-book#19]
Suggestion:
> Provide a little more detail when first encountering steps such as new file creation (e.g. add a new controller). This will obviously be different for VS2017, VS Code and CLI...

**Comment by nbarbettini** [comment]
Creating a new file (for example) is significantly different across VS, VS Code, and other editors, so I avoided the issue the first time around. Not quite sure yet if this adds a ton of extra value, but it may for super-new users?

**Comment by GarethOates** [comment]
Personally I feel like this ventures too far into teaching someone how to use their tools, rather than teaching them the subject at hand.  There's plenty of guides out there already on how to use the various editors and IDEs.  I think adding this might just add a bit too much 'noise'.

**Comment by welkie** [comment]
@GarethOates I agree, but it might make sense to explain how to use at least the "official" tools. Microsoft is betting the farm on VS Code for .NET Core stuff.
